scripts/generate-tests-json.js

```diff
@@ -84,7 +84,7 @@ let run = async () => {
       // Attempt to remove test wrappers but fallback if prettier explodes due to invalid JS
       return format(removeTestWrappers(code))
     } catch (e) {
-      return code
+      return removeTestWrappers(code)
```

Comment:
The only reason format would throw and trigger this code path is if `removeTestWrappers` doesn't generate valid JS, so this isn't safe. A better approach would be to use codemods instead of regex tweaks